GUIDELINES.md

```diff
@@ -0,0 +1,70 @@
+Spider writing guidelines
+=========================
+
+Before start working on a spider, read carefully the following guidelines that will make the project more consistent and your PR more probable to be accepted.
+
+Reading code of existing spiders will also help you to understand how to develop.
```

Comment:
What about adding a link to this?

```suggestion
Reading code of [existing spiders](https://github.com/okfn-brasil/querido-diario/tree/main/processing/data_collection/gazette/spiders) will also help you to understand how to develop.
```

It may change but sometimes it isn't clear where to look when one is starting in a new project.

Reply:
In my opinion, adding links to files in the repository is prone to errors. If we move the file, it is very likely that we will forget to update the docs.